app/scripts/controllers/form.js

```diff
@@ -13,22 +13,16 @@ angular.module('ludwig').controller('FormCtrl', function($scope, $http, $state,
     };
 
     $scope.test.expectedResults.forEach(function(expectedResult) {
-        expectedResult.ref = _.find($scope.possibleValues, { id: expectedResult.code });
```

Comment:
Are you 1000% sure this wasn't necessary anywhere? I've had to write many pieces of code that relied on this `ref` property. I still haven't understood why it's here rather than merged with the values, but I know it is used.